README.md

```diff
@@ -1,30 +1,49 @@
 # SORSE Website repository
 
-This repository contains the source code for the Series of Online Research Software Events 2020. It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes) and hosted with Github Pages. The static website is build automatically on Github but you can also build it locally.
+This repository contains the source code for the Series of Online Research Software
+Events 2020. It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes)
+and hosted with Github Pages. The static website is build automatically on Github
+but you can also build it locally.
 
-The documentation how to contribute is in the [Wiki](wiki) of this repository. There you find, how to add new members, new national chapters, how to assign someone to a specific team, etc.
+The documentation how to contribute is in the [Wiki](https://github.com/SORSE/sorse.github.io/wiki) of this repository.
+There you find, how to add new members, new national chapters, how to assign
+someone to a specific team, etc.
 
 ## How to contribute
 Unless you know exactly what you are doing, please always
 
-- do not push directly to the master branch (usually you are anyway not allowed to do so). You must contribute via pull request
+- do not push directly to the master branch (usually you are anyway not allowed
+  to do so). You must contribute via pull request
 - we always require a review from another person (four-eye-principle)
 - assign issues and pull request to applicable milestones, labels and projects.
-- if your working on an issue or PR, assign it to you! If you are not working on it anymore, remove your assignment from the PR, either because you are finished or because you don't have the time to do more, unassign yourself and ideally leave a message that someone else should take over.
+- if your working on an issue or PR, assign it to you! If you are not working on
```

Comment:
Sorry one more here, “you’re” instead of “your”

Reply:
```suggestion
- if you're working on an issue or PR, assign it to you! If you are not working on
```